.github/workflows/moodle-ci.yml

```diff
@@ -133,9 +133,15 @@ jobs:
 
 
       # 3. Install Atto plugin ( and Quizzes ?)
-      - name: Add MathtType plugin for Atto
+      - name: Add MathType plugin for Atto
+        id: install-plugin-atto
+        continue-on-error: true
         run: |
           moodle-plugin-ci add-plugin --branch ${GITHUB_REF##*/} wiris/moodle-atto_wiris
+      - name: Add MathType plugin for Atto using the stable branch
```

Comment:
Shouldn't we use the `master` branch instead?

Reply:
On the one hand: using `master` is most similar to the workflow we've been following so far, which forced us to create a branch for each project (usually a branch that was sync'd with `master`).

On the other hand: using `stable` is most similar to how a user would experience if they downloaded the branch that we are testing and installed it on their Moodle, as `stable` follows the last released version of the plugins.

I think both make sense. We as a team should come to an agreement.

@icaparros-at-wiris any input?

Reply:
This is a difficult decision 🤔

Using `main` will be great to check if everything works as expected with unreleased changes from other plugins. And will be useful before making a release as will allow us to check all the new changes from the plugins suit.

I also understand the pros of using `stable`. It is great to verify if the filter works with lastest released moodle-atto_wiris plugin.

Depending on the case will be better to use `master` or `stable`, nevertheless, If I have to choose one I prefer to set the `stable`.

Reply:
Okay, will use `stable` then.